plugins/SpectrumAnalyzer/SaWaterfallView.cpp

```diff
@@ -35,6 +35,7 @@
 #include "MainWindow.h"
 #include "SaProcessor.h"
 
+#include <iostream>
```

Comment:
Did you forget to remove that after debugging?

Reply:
Oops, yes, thanks. ^^;